src/services/collectionsTransform.service.js

```diff
@@ -90,12 +91,15 @@ const transformRegForCollections = (registration) => {
     });
 
     operator_keys.forEach((key) => {
```

Comment:
I'm not sure all this is needed. I think if you change the name of the key where it's defined in the operator array at the top to 'operator_companies_house_number' so that it grabs the right value from the data. And then reassign after this forEach statement, i.e. 'operator_company_house_number = operator_companies_house_number' then delete operator_companies_house_number'